index.js

```diff
@@ -284,7 +282,7 @@ async function run(dir) {
   fs.writeFileSync("README.md", readme);
 
   // TODO: Print usage instructions
-  log("Happy BUIDLing!");
+  log("Happy Building! :) ");
```

Comment:
Haha, I can tell you joined the blockchain space after the 2017 hysteria era by just looking at that line change :)

Reply:
is it some old blockchain joke I'm too young to understand? I thought it was a typo :)

Reply:
just googled it...I feel dumb...